tests/kernel/wave/wave_e2e_test.py

```diff
@@ -30,8 +30,10 @@
 ]
 
 user_specified_test_shapes = ""
-
-test_params_path = os.environ.get("TEST_PARAMS_PATH", None)
+default_filename = "test_param.json"

```

Comment:
Please, don't. We need to make all tests work with default shapes.

The core issue that `test_tiled_reduce_max` doesn't work with weird shapes, instead of skipping this shape entirely, we must fix test or mark it xfail.

Reply:
Interesting, so what was the motivation for adding test_param.json shapes to begin with?

Reply:
The motivation was to run perf test with different set of shapes from CI tests.

Reply:
ah, that makes sense, thanks :)

Reply:
I left a comment on https://github.com/iree-org/iree-turbine/pull/119#discussion_r1773607105 . I think pytest marks (for test type / size, as well as xfail) are a better fit here than environment variables and separate json files.

Reply:
@Hardcode84 updated the test ,re-review please :)